README.md

```diff
@@ -444,6 +445,24 @@ Note: For modules use the [@import](http://clang.llvm.org/docs/Modules.html#usin
 #import "NYTUserView.h"
 ```
 
+## Protocols
+
+In a delegate protocol, the first parameter to each method should be the object sending the message.
```

Comment:
Not sure there is an alternative that could be easily grokked. Apple goes with “Delegates and Data Sources”, so that’s what I’ve gone with in the past as well. 

https://developer.apple.com/library/ios/documentation/General/Conceptual/CocoaEncyclopedia/DelegatesandDataSources/DelegatesandDataSources.html